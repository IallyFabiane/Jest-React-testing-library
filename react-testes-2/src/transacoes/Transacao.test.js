import React from 'react';
import { render } from '@testing-library/react';
import Transacao from './Transacao';

describe('Componenete de Transação do Extrato', ()=> {
    it('O snapshot do componente deve premanecer sempre o mesmo', () => {
        const { container } = render(<Transacao 
            data = "08/06/2022"
            tipo = "saque"
            valor = "20.00"
        />)

        expect(container.firstChild).toMatchSnapshot();
    })
})