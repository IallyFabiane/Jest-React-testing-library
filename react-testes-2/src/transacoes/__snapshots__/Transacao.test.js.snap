// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componenete de Transação do Extrato O snapshot do componente deve premanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    08/06/2022
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
